tools/interactive/interactivetool_anylabeling.xml

```diff
@@ -0,0 +1,71 @@
+<tool id="interactive_tool_anylabeling" tool_type="interactive" name="AnyLabeling Interactive" version="@VERSION@" profile="24.2">
+    <description>Effortless data labeling with AI support from YOLO and Segment Anything! AnyLabeling = LabelImg + Labelme + Improved UI + Auto-labeling</description>
```

Comment:
```suggestion
    <description>Effortless data labeling with AI support from YOLO and Segment Anything</description>
```

maybe put that in the help?

Reply:
Do you mind adding an icon to the IT (and others if you like).

```suggestion
    <description>Effortless data labeling with AI support from YOLO and Segment Anything! AnyLabeling = LabelImg + Labelme + Improved UI + Auto-labeling</description>
    <icon src="anylabeling.png" />
```

For example, the icon needs to be next to the XML